koku/api/report/test/oci/test_oci_query_handler.py

```diff
@@ -1538,6 +1538,9 @@ def test_exclude_functionality(self, _):
                         continue
                     opt_dict = opt_dict.get(f"{exclude_opt}s")[0]
                     opt_value = opt_dict.get(exclude_opt)
+                    if "no-" in opt_value:
```

Comment:
This will raise a `TypeError` if the `exclude_opt` key doesn't exists since `opt_value` would be `None`. Defaulting to an empty string will avoid that.
```python
opt_value = opt_dict.get(exclude_opt, "")
```

Reply:
Should this match only be on the beginning of the string? I'm not sure what real values look like but this test would match `some-rhino-sighting`.
```suggestion
                    if opt_value.startswith("no-"):
```

Reply:
👍🏿 glad you pointed this one out